lib/Controller/Folder.php

```diff
@@ -95,7 +95,12 @@ public function grid(Request $request, Response $response, $folderId = null)
             $this->folderFactory->decorateWithSharing($folder);
             $this->folderFactory->decorateWithUsage($folder);
 
-            return $response->withJson($folder);
+            return $response->withJson([
```

Comment:
I don't think this, and the twig changes, should be necessary because unmatched properties should be output when the folder object is JSON serialised

Reply:
Good catch! the issue was there because Folder Entity was not implementing JsonSerializable, therefore the unmatched properties were not on the Folder object - fixed in the commit below